docs/user/alerting/action-types/pagerduty.asciidoc

```diff
@@ -4,19 +4,152 @@
+
+* If you have security enabled:
+** To access the Elastic Alerting functionality, you need
+application privileges to access Metrics, APM, Uptime, or SIEM.
+** If you are using a self-managed deployment with security, you need
+Transport Security Layer (TLS) enabled for communication between Elasticsearch and Kibana.
+Elastic Alerts uses API keys to secure background alert checks and actions,
+and API keys require TLS on the HTTP interface.
+
+Although not a requirement, to harden the integrations security you might want to
+review the <<action-settings, Actions settings>> that are available to you.
+
+[float]
+[[pagerduty-support]]
+=== Support
+If you need help with this integration, get in touch with the Kibana team:
+* Visit https://support.elastic.co[support.elastic.co] or use
+the *Ask Elastic* option in the Kibana Help menu.
+* Select the Kibana option at https://discuss.elastic.co/[discuss.elastic.co].
+
+[float]
+[[pagerduty-integration-walkthrough]]
+=== Integration with PagerDuty walkthrough
+
+[float]
+[[pagerduty-in-pagerduty]]
+==== In PagerDuty
+
+. From the *Configuration* menu, select *Services*.
+. Add an integration to a service:
++
+* If you are adding your integration to an existing service,
+click the name of the service you want to add the integration to.
+Then, select the *Integrations* tab and click the *New Integration* button.
+* If you are creating a new service for your integration,
+go to
+https://support.pagerduty.com/docs/services-and-integrations#section-configuring-services-and-integrations[Configuring Services and Integrations]
+and follow the steps outlined in the *Create a New Service* section, selecting Elastic as the Integration Type in step 4.
+Continue with the <<pagerduty-in-elastic, In Elastic>> section once you have finished these steps.
+
+. Enter an *Integration Name* in the format Elastic-service-name (for example, Elastic-Alerting or Kibana-APM-Alerting)
+and select Elastic from the *Integration Type* menu.
+. Click *Add Integration* to save your new integration.
++
+You will be redirected to the *Integrations* tab for your service. An Integration Key is generated on this screen.
++
+[role="screenshot"]
+image::user/alerting/images/pagerduty-integration.png[PagerDuty Integrations tab]
+
+. Save this key, as you will use it when you configure the integration with Elastic in the next section.
+
+[float]
+[[pagerduty-in-elastic]]
+===== In Elastic
+
+. Create a PagerDuty Connector as described in <<pagerduty-how-it-works, How it works>>.
+
+. Configure the connector by giving it a name and entering the API URL and Routing Key.
++
+*See <<pagerduty-in-pagerduty, In PagerDuty>> for how to obtain the endpoint and key information from PagerDuty and
+<<pagerduty-connector-configuration, Connector configuration>> for additional details.
+
+. Set up an action using your PagerDuty connector, by determining:
```

Comment:
Should we have a step before this that is focused on creating an alert for said action? "Create an alert from the management screen or application of your choice"?